### PR TITLE
color enhancements

### LINKED ISSUE #7: Colorize container selection menu
Container selection should use the same colors as pod selection

### LINKED ISSUE #10: Option to disable colors
If a user's terminal does not jive with colored output there should be an option to disable

### PR DESCRIPTION
**Pull request by gabeduke**
Fixes #10 
Fixes #7

### CODE REVIEW

File: cmd/rsh.go [hunk 1]
```diff
@@ -108,9 +108,19 @@ func (r *iexec) podPrompt(matchingPods []v1.Pod) error {
...
 
 	templates := &promptui.SelectTemplates{
-		Active:   "Namespace: {{ .Namespace | blue }} | Pod: {{ .Name | cyan }}",
+		Active:   fmt.Sprintf("Namespace: {{ .Namespace | blue }} | Pod: %s {{ .Name | cyan }}", promptui.IconSelect),
 		Inactive: "Namespace: {{ .Namespace | blue }} | Pod: {{ .Name | magenta }}",
-		Selected: "Namespace: {{ .Namespace | blue }} | Pod: {{ .Name | cyan }}",
+		Selected: fmt.Sprintf("Namespace: {{ .Namespace | blue }} | Pod: %s {{ .Name | cyan }}", promptui.IconGood),
+	}
+
+	log.Debugf("naked: %v", naked)
```
Comment (zepterfd) [inline cmd/rsh.go]: might be better to move this down one line so it only shows the debug log if naked is true

File: cmd/rsh.go [hunk 2]
```diff
@@ -141,9 +151,26 @@ func (r *iexec) containerPrompt() error {
...
 	}
 
+	templates := &promptui.SelectTemplates{
+		Active:   fmt.Sprintf("Container: %s {{ . | cyan }}", promptui.IconSelect),
+		Inactive: "Container: {{ . | magenta }}",
+		Selected: fmt.Sprintf("Container: %s {{ . | cyan }}", promptui.IconGood),
+	}
+
+	log.Debugf("naked: %v", naked)
+	if naked {
```
Comment (zepterfd) [inline cmd/rsh.go]: how come you have this same if statement repeated as on line 118?

File: README.md [hunk 3]
```diff
@@ -32,9 +32,10 @@ Usage:
   iexec [pod filter] [remote command(s)] [flags]
 
 Flags:
-  -c, --container string   Container to exec into (defaults to container[0]
+  -c, --container string   Container to search
   -h, --help               help for iexec
   -l, --log-level string   log level (trace|debug|info|warn|error|fatal|panic)
+  -x, --naked              Decolorize output
   -n, --namespace string   Namespace to search
   -v, --vimMode            Vim Mode enabled
```
Comment (zepterfd) [inline README.md]: suggestion: change "vimMode" to "vim-mode" or "vimmode" to be consistent with the other descriptive options such as "log-level" and "namespace"

File: cmd/rsh.go (hunk 1 above)
Comment (gabeduke) [reply cmd/rsh.go]: fixed

File: README.md (hunk 3 above)
Comment (gabeduke) [reply README.md]: fixed

File: cmd/rsh.go (hunk 2 above)
Comment (gabeduke) [reply cmd/rsh.go]: We can look at this and discuss what solutions would look like. The `if` conditional is the same but the templates are different.

**Pull request merged**